src/main/kotlin/io/titandata/titan/providers/local/RemoteList.kt

```diff
@@ -13,9 +14,9 @@ class RemoteList (
     
     fun list(container: String) {
         val remotes = remotesApi.listRemotes(container)
-        System.out.printf("%-20s  %s${n}", "REMOTE", "PROVIDER")
+        System.out.printf("%-20s %-20s %s${n}", "REMOTE", "PROVIDER", "URI")
```

Comment:
Do we still need PROVIDER if the first characters of the URI are always the same?

Reply:
Your call. If server will always have a matching uri to provider, then it's redundant. Do you expect this to always be the case?

Reply:
Yes, it's definted to be that way - the prefix before "://" must match the provider exactly.